content/kubeone/master/architecture/supported_providers/_index.en.md

```diff
@@ -13,6 +13,6 @@ the current KubeOne version. The additional features include the KubeOne
 Terraform integration and the integration with Kubermatic machine-controller.
 
 KubeOne supports AWS, Azure, DigitalOcean, GCP, Hetzner Cloud, Nutanix\*,
-OpenStack, Packet, and VMware vSphere.
+OpenStack, Packet, VMware Cloud Director, and VMware vSphere.
```

Comment:
* Please also update the main page (https://docs.kubermatic.com/kubeone/master/)
* Maybe it could be useful to mention when we started supporting Cloud Director (similar to what we did for Nutanix)